test/unit/webdriver/device/system_bars_test.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from test.unit.helper.test_helper import (
+    appium_command,
+    android_w3c_driver,
+    get_httpretty_request_body
+)
+
+import httpretty
+
+
+class TestWebDriverSystemBars(object):
+
+    @httpretty.activate
+    def test_get_system_bars(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.GET,
+            appium_command('/session/1234567890/appium/device/system_bars'),
+            body=''' {"value":
+                     {"statusBar":
+                     {"visible": "True", "x": 0, "y": 0, "width": 1080, "height": 1920},
```

Comment:
Here should be like below. Then, `true` or `false` should not be "True" or "False".

```
{"statusBar":{"visible":true,"x":0,"y":0,"width":1080,"height":63},"navigationBar":{"visible":true,"x":0,"y":0,"width":1080,"height":126}}
```

You can see the log on your appium server log like below.

```
[HTTP] --> GET /wd/hub/session/b757dee1-e1db-4afb-b361-9b58875c1f36/appium/device/system_bars
[HTTP] {}
[debug] [W3C (b757dee1)] Calling AppiumDriver.getSystemBars() with args: ["b757dee1-e1db-4afb-b361-9b58875c1f36"]
[debug] [ADB] Running '/Users/kazu/Library/Android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell dumpsys window windows'
[debug] [W3C (b757dee1)] Responding to client with driver.getSystemBars() result: {"statusBar":{"visible":true,"x":0,"y":0,"width":1080,"height":63},"navigationBar":{"visible":true,"x":0,"y":0,"width":1080,"height":126}}
[HTTP] <-- GET /wd/hub/session/b757dee1-e1db-4afb-b361-9b58875c1f36/appium/device/system_bars 200 67 ms - 148
[HTTP]
```

Reply:
Thanks, I have checked return value ```driver.get_system_bars()```, not value in server log.
I'll see server log from the next time.